install.ps1

```
@@ -122,6 +122,47 @@ function Invoke-InstalledRuntimeFreshnessGate {
     }
   }
 }
+
+function Update-InstallLedgerPayloadSummary {
+  param(
+    [string]$RepoRoot,
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'RepoRoot' has been declared but not used.

(PSReviewUnusedParameter)